### ISSUE TITLE
[JOSS review] Paper updates

### ISSUE THREAD

**Issue by WPettersson** [geoelements/gns#35]
Sorry for renaming this issue, I'll use it to track a few paper issues I have. Also, I will note that the paper is at ~1300 words, while the aim is 750-1000 words - see https://joss.readthedocs.io/en/latest/submitting.html#what-should-my-paper-contain for more.

- [x] State of the art
- [x] GNS training and prediction
- [x] Dataset format
- [x] Parallelisation and scaling
- [x] References
- [x] Contributor guidelines

## State of the art

You state that gns solves trajectory data much faster than MPM simulations. Given that you do use MPM for comparison, you should cite either the technique (probably a textbook), the code you used to generate the comparisons, or preferably both.

## GNS training and prediction

This section could probably also do with some details on what code was used to generate the MPM/SPH training data.

## Dataset format

This section isn't necessary for the paper (although would be useful for some sort of online documentation)

## Parallelisation and scaling

You don't explain what `a_t^i` is, and you should be consistent between `x_i` and `x_t^i` (in the first case, i is time, but in the second, t is time and i is particle index I think?). However, I also don't know what this is meant to add to the paper, as all the formula says is that you are averaging over particles to get loss. The exact technique for parallelisation also isn't necessary.

## References

* Graph Network Simulator appears twice. It seems like one of the links is for the data, not for the software itself. Also the author order doesn't agree on one of them (K. Kumar is listed first according to both links, but J. Vantassel appears first in one of the entries in the references section).
* At least one article is missing a DOI according to the bot, but I think a few others are also missing - I found https://dl.acm.org/doi/10.5555/3524938.3525722 for "Learning to simulate complex physics with graph networks - so please check all of your references for DOIs.

## Contributor guidelines

I see you have you have issue templates which are good, and a Contributing.md file, which is also good, but the contributing.md file doesn't seem to have much instructions for people wanting to support your work (i.e., what branch should pull requests be made against, style choices, any particular formatting requirements for commit messages). I personally use, and often recommend to others, something like https://contributing.md/example/ but altered with whatever changes you wish to make.

**Comment by osorensen** [comment]
Nice to see that you opened issues here @WPettersson. I just add a link to the review here so we keep track: https://github.com/openjournals/joss-reviews/issues/5025

**Comment by kks32** [comment]
@WPettersson Thank you. I have made the requested changes.

**Comment by WPettersson** [comment]
Looks good to me :)
